Cargo.toml

```diff
@@ -11,10 +11,13 @@ license = "AGPL-3.0-only"
 
 [dependencies]
 # for local development, you can use a path dependency
-nu-plugin = { path = "../nushell/crates/nu-plugin", version = "0.95.1" }
-nu-protocol = { path = "../nushell/crates/nu-protocol", version = "0.95.1" }
-# nu-plugin = "0.80.0"
-# nu-protocol = "0.80.0"
+# nu-plugin = { path = "../nushell/crates/nu-plugin", version = "0.95.1" }
+# nu-protocol = { path = "../nushell/crates/nu-protocol", version = "0.95.1" }
+
+# people are likely to try and `git clone` -> `cargo install` this.
+# it is better left as the currently avalible crates.io package
+nu-plugin = "0.95.0"
+nu-protocol = "0.95.0"
```

Comment:
I'd prefer these to stay the way I had them please, with 0.95.1

Reply:
current nu-plugin/protocol on crates.io is 0.95.0. with the previous Cargo.toml compilation failed with a weird non-descriptive error. i argue that this is better since it will compile without any effort by an end user who may not understand rust, cargo, etc. i will change it back but ask you to reconsider for the sake of the end user

Reply:
Yup, I get your point but the reason that I left the nu-plugin and nu-protocol part commented out is so people like you could easily change it to 0.95.0 and compile if they wanted to. In order to get this PR landed, it'll have to be set back the way I originally had it. Thanks